<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_PodchezertsevClass</class>
 <widget class="QMainWindow" name="Qt_PodchezertsevClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt_Podchezertsev</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>153</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Add connector</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_male">
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_female">
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="placeholderText">
        <string>Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_connector">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>153</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Add device</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_len">
       <property name="placeholderText">
        <string>Max lenght</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_device">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>221</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Devices</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>221</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Active connectors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_merge">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge two last</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Qt_Podchezertsev.qrc"/>
 </resources>
 <connections/>
</ui>
